pkgs/applications/radio/qdmr/default.nix

```diff
@@ -0,0 +1,68 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeDesktopItem,
+  writeText,
+  cmake,
+  wrapQtAppsHook,
+  libusb1,
+  libyamlcpp,
+  libsForQt5,
+  qtlocation,
+  qtserialport,
+  qttools,
+  qtbase,
+}:
+stdenv.mkDerivation rec {
+  pname = "qdmr";
+  version = "0.11.0";
+
+  src = fetchFromGitHub {
+    owner = "hmatuschek";
+    repo = "qdmr";
+    rev = "v${version}";
+    sha256 = "sha256-N8s2Qva9PNu8fXcMNpLnhYpVgol58Ieh4vC95YJw0Yc=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+    wrapQtAppsHook
+  ];
+
+  buildInputs = [
+    libyamlcpp
+    libusb1
+    libsForQt5.qt5.qtlocation
```

Comment:
with the change above, you can also drop `libForQt5` here

Reply:
It will only build if i remove libsForQt5.qt5 but dosn't that stop ensuring that it ueses qt5 libs instead of some other version, or is there some nixos magick going on making it know it hase to use qt5?

Reply:
There's only a qt5 version of `qtlocation`. Maybe also qt6, you'd get that if you were using `qt6Packages.callPackage` instead of `libsForQt5.callPackage`.

Reply:
Okay then I will do that, is there anything else I can improve? Also the developer of qdmr confirmed that the segfault issue when writing to the radio is a bug in qdmr. https://github.com/hmatuschek/qdmr/issues/275 and I can confirm that the patch fixed the issue